frappe/commands/utils.py

```diff
@@ -36,6 +36,12 @@
 	default=False,
 	help="Saves esbuild metafiles for built assets. Useful for analyzing bundle size. More info: https://esbuild.github.io/api/#metafile",
 )
+@click.option(
+	"--using-cached",
+	is_flag=True,
+	default=False,
+	help="Skips build and uses cached build artifacts (cache is set by Bench). Ignored if developer_mode enabled.",
+)
```

Comment:
Add an environment variable for this too. Otherwise, you won't be able to use this without knowing the exact version of Framework where it's supported.

Reply:
I'm confused, do you mean FF should set an environment variable to signal Bench that `using-cached` is available in FF?

Reply:
Since environment variables wouldn't have worked here (for frappe to signal bench that it can use cached), I've added a (dummy) command to do it.
- `frappe`: https://github.com/frappe/frappe/pull/24412/commits/c7ecd2454ef8ef35239501392eb7b92fdb74a1b4
- `bench`: https://github.com/frappe/bench/pull/1519/commits/d177d8ff633fe04a685754d7b8f52a19a8e8c2bf

Reply:
Click allows setting args from environment. It's much simpler option IMO.

https://github.com/frappe/frappe/blob/53d4153bcda574ae5017b5ace1c1ff79bca3e028/frappe/commands/utils.py#L26

Reply:
Yep (wasn't aware), cleaner and quicker too.

Although, since it's less direct kinda hard to miss. And in this case works cause bench's functionality need not change dependent on frappe's support for _using cached_.

Changes:
- `frappe`: https://github.com/frappe/frappe/pull/24412/commits/2da12b27cd054a11bcb8e657764bb27524656ace
- `bench`: https://github.com/frappe/bench/pull/1519/commits/683a421e43d04a4ef6ea11f95f2ce562bbafcfca